core/Plugin/Controller.php

```diff
@@ -947,64 +948,6 @@ public static function getPrettyDate($date, $period)
         return self::getCalendarPrettyDate(Period\Factory::build($period, Date::factory($date)));
     }
 
-    /**
-     * Calculates the evolution from one value to another and returns HTML displaying
-     * the evolution percent. The HTML includes an up/down arrow and is colored red, black or
-     * green depending on whether the evolution is negative, 0 or positive.
-     *
-     * No HTML is returned if the current value and evolution percent are both 0.
-     *
-     * @param string $date The date of the current value.
-     * @param int $currentValue The value to calculate evolution to.
-     * @param string $pastDate The date of past value.
-     * @param int $pastValue The value in the past to calculate evolution from.
-     * @return string|false The HTML or `false` if the evolution is 0 and the current value is 0.
-     * @api
-     */
-    protected function getEvolutionHtml($date, $currentValue, $pastDate, $pastValue)
```

Comment:
Was only used in Referrers\Controller. Now implemented by `Sparklines` visualization